Core/Rewriters/ObsoleteMethodRewriter.cs

```diff
@@ -36,13 +36,8 @@ public ObsoleteMethodRewriter (IFormatter formatter)
       {
         trackedNodes = node.TrackNodes (node.DescendantNodesAndSelf(), CompilationId);
       }
-      catch (Exception ex)
+      catch (Exception)
```

Comment:
No more warning?

Reply:
No because this has nothing to do with the node. Sometimes `TrackNodes` throws a `KeyNotFoundException` or an `ArgumentException`, while tracking the nodes.

Reply:
The node still can't be transformed right? Might still be good to add a warning that states this, even though it would lead to a compile error.